proto/sifnode/dispensation/v1/query.proto

```diff
@@ -37,4 +39,11 @@ message QueryRecordsByRecipientAddrRequest { string address = 1; }
 message QueryRecordsByRecipientAddrResponse {
   DistributionRecords distribution_records = 1;
   int64 height = 2;
+}
+
+message QueryClaimsByTypeRequest { DistributionType user_claim_type = 1; }
```

Comment:
Is user_claim_type a good name here? Isn't claim_type / distribution_type sufficient?